SpecflowAPITests/Steps/RequestStepDefinitions.cs

```diff
@@ -80,6 +80,13 @@ public void WhenIStoreProjectIdForWorkspaceCleaning()
             _helper.StoreId(jsonObject.SelectToken("id").ToString());
         }
 
+        [When(@"I store board id for workspace cleaning")]
```

Comment:
Created another definition for board.
Because I didnt want to touch project features files, but one option could be merge both as just one.